codalab/lib/cli_util.py

```diff
@@ -62,7 +62,8 @@ def nested_dict_get(obj, *args, **kwargs):
 
 def parse_key_target(spec):
     """
-    Parses a keyed target spec into its key and the rest of the target spec
+    Parses a keyed target spec into its key and the rest of the target spec.
+    Raise UsageError when the value of the target spec is empty.
     :param spec: a target spec in the form of
         [[<key>]:][<instance>::][<worksheet_spec>//]<bundle_spec>[/<subpath>]
```

Comment:
Should this be `[[<key>]:]` (RUN_TARGET_SPEC_FORMAT) or `[[<key>:]]` (MAKE_TARGET_SPEC_FORMAT)? What's the difference?